### DISCUSSION TITLE
CPU usage is 100% on PI3B+

### DISCUSSION THREAD

**Discussion by tomm2001** [raspiblitz/raspiblitz#4550] [Q&A]
Please tell me, is it normal when the processor is loaded at 100%? I can't even connect to raspberry via the terminal. How can I limit resource consumption by processes?
![1](https://github.com/raspiblitz/raspiblitz/assets/101186077/7e9becaf-480e-4c46-8608-fb0a6117975d)
![2](https://github.com/raspiblitz/raspiblitz/assets/101186077/51270922-68a9-45b1-8f87-34d3fb1ad36a)

**Comment by rootzoll** [comment]
Have to say that we dont test anymore and RaspberryPi3 ... Pi 4&5 are the recommended models now.
